storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6398,7 +6397,7 @@ static bool xb_export_cfg_write_index_fields(
     mach_write_to_4(value, table->first_index()->n_instant_nullable);
 
     if (fwrite(&value, 1, sizeof(value), file) != sizeof(value)) {
-      msg("xtrabackup: Error: writing table meta-data.");
+      xb::error() << "Error writing table meta-data.";
```

Comment:
```suggestion
      xb::error() << "writing table meta-data.";
```

Reply:
Althoug we have error prefix, it may seem odd to just see "writing table meta-data" So I intentionally retained this error

Reply:
or append failed in the end:

`xb::error() << "writing table meta-data failed.";`